bowbuffer.gen.go.tmpl

```diff
@@ -32,7 +47,7 @@ func (b *Buffer) Len() int {
 	}
 }
 
-func (b *Buffer) SetOrDrop(i int, value interface{}) {
+func (b Buffer) SetOrDrop(i int, value interface{}) {
```

Comment:
I find it tricky that a Set method does not expect a pointer receiver

Reply:
it were bugging me too, at first. But after a while it's interesting to know those are all slice and nothing mutate on headers after a constructor call, only underlying memory.
Series could be this way too.
I can reverse this. it matter after all since it copies headers on stack on each call instead of reading a copied pointer.
I'll benchmark it for fun first and give you the result.